[BUG] 无法通过编译

### Self-check / 自查

- [x] DNSMASQ's `DNS Redirect` option is turned off (ignore if not applicable) / 已经关闭了DNSMASQ的`DNS 重定向`选项（如无此项请忽略）
- [x] The other proxy app has been stopped and the environment is normal / 已停止运行其他代理插件，并确认环境正常

### Confirm / 确认

- [x] I have checked the Wiki and found no solution / 我已经查看过Wiki，没有找到解决办法
- [x] I have searched the Issue and found no related issues / 我已经搜索过Issue，没有找到相关问题
- [x] I am using the latest version of the app which is build from this repository / 我使用的是本仓库构建的最新版的插件
- [x] I provided information which does not include sensitive information / 我提供的信息里不包含敏感信息
- [x] I have provided correct, valid, and helpful information for debugging / 我提供了正确的、有效的、可以帮助DEBUG的信息

### BUG Description / BUG 描述

1.26.0版本编译失败

### Expected Behavior / 预期行为

编译成功

### Reproduction Steps / 复现步骤

make package/feeds/custom_feed/nikki/compile V=s

### Debug Log / 调试日志

CONFIG_GOLANG_MOD_CACHE_WORLD_READABLE="" GO_BUILD_CACHE_DIR="/home/wh/ipq_wrt_relese/imm-nss/tmp/go-build" GO_MOD_CACHE_DIR="/home/wh/ipq_wrt_relese/imm-nss/dl/go-mod-cache" GO_MOD_ARGS="-modcacherw" GO_PKG="github.com/metacubex/mihomo" GO_INSTALL_EXTRA="" GO_INSTALL_ALL="" GO_SOURCE_ONLY="" GO_BUILD_PKG="github.com/metacubex/mihomo/..." GO_EXCLUDES="" GO_GO_GENERATE="" GO_INSTALL_BIN_PATH="/usr/bin" BUILD_DIR="/home/wh/ipq_wrt_relese/imm-nss/build_dir/target-aarch64_cortex-a53_musl/nikki-2026.04.08" GO_BUILD_DIR="/home/wh/ipq_wrt_relese/imm-nss/build_dir/target-aarch64_cortex-a53_musl/nikki-2026.04.08/.go_work/build" GO_BUILD_BIN_DIR="/home/wh/ipq_wrt_relese/imm-nss/build_dir/target-aarch64_cortex-a53_musl/nikki-2026.04.08/.go_work/build/bin/linux_arm64" GO_BUILD_DEPENDS_PATH="/usr/share/gocode" GO_BUILD_DEPENDS_SRC="/home/wh/ipq_wrt_relese/imm-nss/staging_dir/target-aarch64_cortex-a53_musl/usr/share/gocode/src" /usr/bin/env bash /home/wh/ipq_wrt_relese/imm-nss/feeds/packages/lang/golang//golang-build.sh install_src "/home/wh/ipq_wrt_relese/imm-nss/tmp/stage-nikki"

cp: cannot stat '/home/wh/ipq_wrt_relese/imm-nss/build_dir/target-aarch64_cortex-a53_musl/nikki-2026.04.08/.go_work/build/src/github.com/metacubex/mihomo': No such file or directory

make[2]: *** [Makefile:94: /home/wh/ipq_wrt_relese/imm-nss/staging_dir/target-aarch64_cortex-a53_musl/stamp/.nikki_installed] Error 1

make[2]: Leaving directory '/home/wh/ipq_wrt_relese/imm-nss/custom_feed/nikki'

time: package/feeds/custom_feed/nikki/compile#0.86#1.75#2.56

    ERROR: package/feeds/custom_feed/nikki failed to build.

make[1]: *** [package/Makefile:188: package/feeds/custom_feed/nikki/compile] Error 1

make[1]: Leaving directory '/home/wh/ipq_wrt_relese/imm-nss'

make: *** [/home/wh/ipq_wrt_relese/imm-nss/include/toplevel.mk:233：package/feeds/custom_feed/nikki/compile] 错误 2

### App Log / 插件日志

无

### Core Log / 核心日志

无

### Additional Information / 附加信息

_No response_

--- a/mihomo-alpha/Makefile
+++ b/mihomo-alpha/Makefile
@@ -21,6 +21,7 @@ PKG_BUILD_VERSION:=alpha-c59c99a0
 PKG_BUILD_TIME:=$(shell date -u -Iseconds)
 
 GO_PKG:=github.com/metacubex/mihomo
+GO_PKG_BUILD_PKG:=github.com/metacubex/mihomo
 GO_PKG_LDFLAGS_X:=$(GO_PKG)/constant.Version=$(PKG_BUILD_VERSION) $(GO_PKG)/constant.BuildTime=$(PKG_BUILD_TIME)
 GO_PKG_TAGS:=with_gvisor
 GO_PKG_INSTALL_BIN_PATH:=/usr/libexec

--- a/mihomo-meta/Makefile
+++ b/mihomo-meta/Makefile
@@ -21,6 +21,7 @@ PKG_BUILD_VERSION:=v1.19.24
 PKG_BUILD_TIME:=$(shell date -u -Iseconds)
 
 GO_PKG:=github.com/metacubex/mihomo
+GO_PKG_BUILD_PKG:=$(GO_PKG)
 GO_PKG_LDFLAGS_X:=$(GO_PKG)/constant.Version=$(PKG_BUILD_VERSION) $(GO_PKG)/constant.BuildTime=$(PKG_BUILD_TIME)
 GO_PKG_TAGS:=with_gvisor
 GO_PKG_INSTALL_BIN_PATH:=/usr/libexec

--- a/mihomo-meta/Makefile
+++ b/mihomo-meta/Makefile
@@ -21,7 +21,7 @@ PKG_BUILD_VERSION:=v1.19.24
 PKG_BUILD_TIME:=$(shell date -u -Iseconds)
 
 GO_PKG:=github.com/metacubex/mihomo
-GO_PKG_BUILD_PKG:=$(GO_PKG)
+GO_PKG_BUILD_PKG:=github.com/metacubex/mihomo
 GO_PKG_LDFLAGS_X:=$(GO_PKG)/constant.Version=$(PKG_BUILD_VERSION) $(GO_PKG)/constant.BuildTime=$(PKG_BUILD_TIME)
 GO_PKG_TAGS:=with_gvisor
 GO_PKG_INSTALL_BIN_PATH:=/usr/libexec

